test/utils/form.ts

```diff
@@ -0,0 +1,296 @@
+    this.page = page;
+  }
+
+  public get() {
+    return this.locator;
+  }
+
+  public async setValue(fieldKey) {
+    await this.get().click();
+    return this.page.getByGrafanaSelector(this.page.ctx.selectors.components.Select.option).getByText(fieldKey).click();
+  }
+}
+
+/**
+ * Elements Helper
+ */
+class ElementsHelper {
+  public selectors: LocatorSelectors<typeof TEST_IDS.formElements>;
+
+  constructor(public readonly locator: Locator) {
+    this.selectors = this.getSelectors(locator);
+  }
+
+  private getMsg(msg: string): string {
+    return `Elements: ${msg}`;
+  }
+
+  private getSelectors(locator: Locator) {
+    return getElementsSelector(locator);
+  }
+
+  public async checkPresence() {
+    return expect(this.selectors.root(), this.getMsg('Elements Container Presence')).toBeVisible();
+  }
+
+  public async checkElementPresence(elementId: string, elementType: string) {
+    return expect(
+      this.selectors.element(elementId, elementType),
+      this.getMsg(`Element ${elementId} Presence`)
+    ).toBeVisible();
+  }
+
+  public async checkElementNotPresence(elementId: string, elementType: string) {
+    return expect(
+      this.selectors.element(elementId, elementType),
+      this.getMsg(`Element ${elementId} not Presence`)
+    ).not.toBeVisible();
+  }
+
+  public async getElement(elementId: string, elementType: string) {
+    return this.selectors.element(elementId, elementType);
+  }
+
+  public async getDisabledElement(elementId: string, elementType: string) {
+    const element = await this.getElement(elementId, elementType);
+    return new DisabledElementHelper(element);
+  }
+
+  public async getNumberElement(elementId: string, elementType: string) {
+    const element = await this.getElement(elementId, elementType);
+    return new NumberElementHelper(element);
+  }
+
+  public async getSelectElement(elementId: string, elementType: string, dashboardPage: DashboardPage) {
+    const element = await this.getElement(elementId, elementType);
+    return new SelectElementHelper(element, dashboardPage);
+  }
+}
+
+/**
+ * Submit Button Helper
+ */
+class SubmitButtonHelper {
+  private readonly locator: Locator;
+
+  constructor(parentLocator: Locator) {
+    this.locator = parentLocator.getByTestId(TEST_IDS.panel.buttonSubmit);
+  }
+
+  private getMsg(message: string): string {
+    return `Submit Button: ${message}`;
+  }
+
+  public get() {
+    return this.locator;
+  }
+
+  public async checkPresence() {
+    return expect(this.get(), this.getMsg(`Check Submit button Presence`)).toBeVisible();
+  }
+
+  public async checkIsDisabled() {
+    return expect(this.get(), this.getMsg(`Check Submit button disabled`)).toBeDisabled();
+  }
+
+  public async checkIsNotDisabled() {
+    return expect(this.get(), this.getMsg(`Check Submit button not disabled`)).not.toBeDisabled();
+  }
+
+  public async submit() {
+    return this.get().click();
+  }
+}
+
+/**
+ * Reset Button Helper
+ */
+class ResetButtonHelper {
```

Comment:
Do we really need helper for each button? can it be done by one helper - ButtonsHelper or move methods to the panel helper